packages/@react-stately/data/test/useAsyncList.test.js

```diff
@@ -40,6 +40,10 @@ describe('useAsyncList', () => {
     jest.useFakeTimers();
   });
 
+  afterEach(async () => {
+    await act(async () => jest.runAllTimers());
+  });
+
```

Comment:
seemed to help the build, sometimes it failed this test due to useAsyncList dispatches being called after the test finished